doc/tutorial.rst

```diff
@@ -30,11 +30,11 @@ Charliecloud for your own applications.
 
 ::
 
-  $ cd /usr/local/src/charliecloud/examples/serial/hello
+  $ cd /usr/local/share/doc/charliecloud/examples/hello
   $ ch-build -t hello .
   Sending build context to Docker daemon  5.632kB
   [...]
-  Successfully built 1136de7d4c0a
+  Successfully built eb5f42d5bb54
```

Comment:
FWIW, I don't bother to update hashes in the docs. I don't think you should change it back though.